content/doc/index.adoc

```diff
@@ -12,8 +12,7 @@ standalone by any machine with the Java Runtime Environment installed.
 == Guided Tour
 
 This guided tour will use the "standalone" Jenkins distribution which requires
-a minimum of Java 7, though Java 8 is recommended. A system with more than
-512MB of RAM is also recommended.
+a minimum of Java 8. A system with more than 512MB of RAM is also recommended.
```

Comment:
Given that we're not yet ready for the soon-to-be-released Java 9, could we rephrase it as "which requires Java 8"?